mkdocs.yml

```
@@ -599,8 +600,8 @@ nav:
           - organizations/using-gate-policies.md
           - organizations/using-coding-standards.md
           - organizations/copying-code-patterns-between-repositories.md
-          - organizations/configuring-default-git-provider-integration-settings.md
           - Integrations:
```

What if we take the chance to use a gerund here, for information architecture consistency? 🤔 

```suggestion
          - Managing integrations:
```

---

Good idea. I also renamed the one under the "Configuring your repositories" section.